src/player.h

```diff
@@ -247,6 +247,9 @@ class Player
 	u32 hud_flags;
 	s32 hud_hotbar_itemcount;
 
+	// Get proper max HUD itemcount
```

Comment:
```suggestion
	// Get actual usable number of hotbar items (clamped to size of "main" list)
```

Reply:
could also remove `getHotbarItemcount` to make it clearer that it's not intended way of accessing it

Reply:
I see getHotbarItemcount using somewhere this internal value is needed, so I am keeping it.

Reply:
Some incorrect use of getHotbarItemcount are fixed in 2f5a13f0e